plugins/resources/app/javascript/app/components/autoscaling/config_ng/subscopes.js

```diff
@@ -276,10 +276,11 @@ const AutoscalingConfigNgSubscopes = ({
     return items.sort((a, b) =>
       a.service < b.service ? -1 : a.service > b.service ? 1 : 0
     );
-  }, [subscopesState, searchTerm]);
+  }, [JSON.stringify(subscopesState), searchTerm]);
 
   const editModeCount = React.useMemo(
-    () => filteredSubscopes.filter((s) => s.editMode).length
+    () => filteredSubscopes.filter((s) => s.editMode).length,
+    [JSON.stringify(filteredSubscopes)]
   );
```

Comment:
Why is this code (both the existing and the new code) using `useMemo` for everything here? I'm pretty sure that you're using a more expensive operation (`JSON.stringify`) to save work on a simpler operation (`Array.filter`).